api/spec/requests/spree/api/checkouts_controller_spec.rb

```diff
@@ -7,13 +7,13 @@ module Spree
     before(:each) do
       stub_authentication!
       Spree::Config[:track_inventory_levels] = false
-      country_zone = create(:zone, name: 'CountryZone')
+      @country_zone = create(:zone, name: 'CountryZone')
```

Comment:
I know there's no easy way to get these zones set up correctly with factories and we should give a simpler interface around this but in the meantime I'd rather start moving these `ivar` into `let!` instead of adding new ones. Do you think it's doable here?

Reply:
Yeah, let me give it shot.